frontend/src/App.jsx

```diff
@@ -12,7 +17,13 @@ function App() {
     <Router>
 
       <Routes>
-        <Route path="/" element={<HomePage />} />
+        <Route path="/" element={<InitialPage />} />
+        <Route path="/SignUp" element={<SignupPage />} />
+        <Route path="/Home" element={<HomePage />} />
```

Comment:
I think some thoughts can be put into user flow here. 
Generally when we land on a website, we don't want to first see a plain login page. We want to see some public content that gives us a sense of what the website offers etc. If we have logged in and cookie hasn't expired, we should have logged in. If not, we can choose to log in or sign up from the nav bar.

Reply:
If you decide to address the user flow comment later, please add TODO in a comment so that it doesn't left forgotten. Add something in code like this:
`// TODO: login/signup routing logic

Reply:
To do

Reply:
Done